Artifact name fixes as discussed in #26.

lsm5 opened the pull request:
PR #26 contained some changes for fixing artifact names and getting rid of source files in the final artifact list. This PR incorporates those changes into the main branch.

<!--
Thanks for sending a pull request!

Please make your commit messages insightful so we start a decent history. If you
add something to the main Containerfile, please ensure you add a test in `verify/`
so we can protect against regressions.
-->

<!--
If no, just write `None` in the release-note block below. If yes, a release note
is required: Enter your extended release note in the block below. If the PR
requires additional action from users switching to the new release, include the
string "action required".

For more information on release notes, please follow the Kubernetes model:
https://git.k8s.io/community/contributors/guide/release-notes.md
-->

```release-note
None
```

baude commented:
is the `stage` prefix expected?

lsm5 commented:
> is the `stage` prefix expected?

That's how it used to be on main branch. Should that be changed to just `podman-machine` ?

lsm5 commented:
@baude on aarch64, the tests seem to be failing in rootless mode https://artifacts.dev.testing-farm.io/71e92fbd-56a2-4f0e-ac85-4b5a58b03d02/ . Should I try switching to rootful ? x86_64 failures are a known infra problem that I think is being worked on.

Luap99 commented:
podman machine will not run as root, so we must run the verify suite rootless.

Luap99 commented on util.sh:
```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Install deps
+dnf -y install jq dosfstools
+
```
can you drop the install here, this was added in https://github.com/containers/podman-machine-os/pull/34

lsm5 replied:
done

Luap99 reviewed (commented):
Naming changes LGTM but please drop the dnf install from the script

lsm5 committed:
Artifact name fixes as discussed in #26.

PR #26 contained some changes for fixing artifact names and getting rid
of source files in the final artifact list. This PR incorporates those
changes into the main branch.

Additionally, reduce the envvars in podman-rpm-info-vars.sh and get them from the
container itself.

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

lsm5 commented:
@Luap99 @baude PTAL

Luap99 reviewed (approved):
LGTM
